examples/tutorials/02_pipeline_operations.ipynb

```diff
@@ -15,7 +15,6 @@
    "source": [
```

Comment:
<blockquote>Sometimes, you need to do something only once before/after the main pipeline flow. This is where <code>before</code> and <code>after</code> pipelines come in, providing a convenient way of making one-off actions. For example, you can use <code>before</code> pipeline to initialize all the variables and models and <code>after</code> to save model and evaluate metrics.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>